homeassistant/components/media_player/__init__.py

```diff
@@ -1089,3 +1079,13 @@ def as_dict(self, *, parent: bool = True) -> dict:
             response["children"] = []
 
         return response
+
+    def calculate_children_class(self):
+        """Count the children media classes and calculate the correct class."""
+        self.children_media_class = MEDIA_CLASS_DIRECTORY
+        if len(self.children) == 0:
+            return
```

Comment:
Let's only change the default if there are children.
```suggestion
        if len(self.children) == 0:
            return
        
        self.children_media_class = MEDIA_CLASS_DIRECTORY
```